README.md

```diff
@@ -1,16 +1,309 @@
 
-## App Highlights
+Whether you're following a textbook, taking a class, or working with a tutor, Sentence Studio lets you import your specific vocabulary lists and practice the exact grammar patterns you need to master. This targeted approach ensures your practice time directly supports your primary learning goals.
 
-- OpenAI integration for conversation, creating puzzles, analyzing photos, scoring language usage
-- Scriban for prompt text templates
-- SQLite data store
-- Desktop and Mobile UX
-- Fluent design patterns
-- MVVM with .NET Community Toolkit
+## Overview
 
-## SQLite
+Sentence Studio leverages artificial intelligence to create engaging language learning experiences across multiple platforms. Whether you're a beginner building foundational skills or an advanced learner refining your command of the language, the application provides structured practice opportunities that mirror effective classroom instruction methods.
 
-This app stores data in SQLite. For viewing and editing the database on Mac, I'm using [sqlitebrowser](https://sqlitebrowser.org).
\ No newline at end of file
+### Bring Your Own Curriculum
+
+The power of Sentence Studio lies in its ability to work with your existing learning materials. You can import:
+
+- **Custom Vocabulary Lists**: CSV files with your specific vocabulary from textbooks, classes, or personal study
+- **Grammar Focus Areas**: Text files outlining the specific grammar patterns you're working on
+- **Themed Content**: Vocabulary and exercises organized around topics relevant to your studies
+
+#### Example Usage
+
+For a lesson focused on daily routines using present simple tense, you might create:
+
+**Vocabulary File ([docs/examples/Lesson1-vocab.csv](docs/examples/Lesson1-vocab.csv)):**
+```csv
+English,Translation,Category,Example Sentence
+wake up,despertarse,Daily Routine,I wake up at 7 AM every day.
+breakfast,desayuno,Daily Routine,I eat breakfast with my family.
+work,trabajar,Daily Activities,She works in an office downtown.
+study,estudiar,Daily Activities,He studies English every evening.
+```
+
+**Skills File ([docs/examples/Lesson1-skills.txt](docs/examples/Lesson1-skills.txt)):**
+```
+Grammar Focus: Present Simple Tense
+- Forming positive statements with regular verbs
+- Using "be" verb forms (am, is, are)
+- Basic sentence structure: Subject + Verb + Object
+
+Communication Goals:
+- Introducing yourself and others
+- Describing daily habits and routines
+- Expressing preferences and dislikes
+```
+
+This targeted approach ensures your practice sessions directly reinforce what you're learning in your primary course or textbook.
+
+### Core Learning Features
```

Comment:
add a couple of images to highlight the app experience.

<img width="960" height="769" alt="image" src="https://github.com/user-attachments/assets/0a0ed8ca-b6f5-4083-82d8-d31cb635236a" />
<img width="960" height="769" alt="image" src="https://github.com/user-attachments/assets/632c6972-90be-4fb0-91ab-29a4a9e8e94e" />
<img width="960" height="769" alt="image" src="https://github.com/user-attachments/assets/363f41a4-16ef-46d5-b9cc-9d6cdbf42cc4" />